.travis.yml

```diff
@@ -85,6 +85,7 @@ script:
       else
         py.test -m "not avoid_travis" distributed $PYTEST_OPTIONS;
       fi;
+    - flake8 distributed/*.py distributed/{bokeh,protocol,deploy}/*.py # no tests yet
```

Comment:
Probably.  I also think that we should eventually include tests.  I think that we should aim to change this to `flake8 distributed` in the near future.  I wanted to finish what I had done so far and emit a PR though (I probably won't work on this much more today and don't want a PR like this growing stale (which will happen quickly))